modules/nw-dual-stack-convert.adoc

```diff
@@ -4,7 +4,7 @@
 
 As a cluster administrator, you can convert your single-stack cluster network to a dual-stack cluster network. After converting to a dual-stack network, new and existing pods have dual-stack networking enabled.
 
-Converting a single-stack cluster network to a dual-stack cluster network consists of creating patches and applying them to the cluster's network and infrastructure.
+Converting a single-stack cluster network to a dual-stack cluster network consists of creating patches and applying them to the cluster's network and infrastructure. You can convert to a dual-stack cluster network for a cluster that runs on either installer-provisioned infrastructure or user-provisioned infrastructure.
```

Comment:
@rbbratta do you know UPI is tested before?  
I guess we just need to specified IPI @dfitzmau

Reply:
Dual-stack has not been tested on vSphere UPI and is not supported.

I don't know about baremetal UPI.

Reply:
Yes, +1,  so I guess we just need mentioned IPI for this feature

Reply:
Fair, from my side I have to agree with QE here - if we do not officially test this combination, then we should be careful with exposing it in docs